src/components/Pagination/Pagination.tsx

```diff
@@ -2,0 +1,79 @@
+import React from 'react';
+
+type Props = {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (page: number) => void;
+};
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage = 1,
+  onPageChange,
+}) => {
+  const totalPages = Math.ceil(total / perPage);
```

Comment:
The calculation of totalPages using Math.ceil(total / perPage) is correct. This ensures that any remaining items that don't fill a complete page are accounted for in an additional page.